src/ca/team2706/vision/trackerboxreloaded/Main.java

```diff
@@ -78,26 +137,29 @@ private static BufferedImage Mat2BufferedImage(Mat matrix) throws Exception {
 		return bi;
 	}
 
-	public static void main(String[] args) {
+	/**
+	 * The main method!
+	 * Very important
+	 * Do not delete!
+	 * :]
+	 * :]
+	 * @param args
+	 * @throws Exception
+	 */
+	public static void main(String[] args) throws Exception {
 		new Main();
 	}
-
-	public Main() {
+	/**
```

Comment:
uhh, nope.

1. I'm uncomfortable with being able to create multiple copies of Main() ... that doesn't really make sense ... also, it'll probably crash if we create two copies of Main() when the second one tries to grab the camera. Why did you do this, was there a problem you were trying to solve with this?
2. `main(..) throws Exception` is super lazy. Exceptions are the seat belts of programming, you can't just get rid of them because they get in your way. Hacks like this to bypass proper error handling is what leads to buggy, crashy code.

Please fix.

Reply:
@ounsworth i just pushed a commit that fixed these